frontend/components/PlatformSelector/PlatformSelector.tsx

```diff
@@ -11,6 +12,7 @@ interface IPlatformSelectorProps {
   setCheckWindows: (val: boolean) => void;
   setCheckLinux: (val: boolean) => void;
   setCheckChrome: (val: boolean) => void;
+  disabled?: boolean;
```

Comment:
Added optional disabled value used when showing disabled view when fetching the autofill API [Figma design](https://www.figma.com/file/K3MckCC9bv4fXMd1mHeFFA/%2318187-AI-generated-calendar-description-for-policy-remediation?type=design&node-id=2-130&mode=design&t=guYAiSOn5cSa8A2L-0)